modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -25,6 +25,7 @@
  */      
 class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
 {
+    var $AjaxModule = true;
```

Comment:
I think it's fine to be consistent with the existing codebase, even if we change it later.